src/main/harmonysolver.qml

```diff
@@ -458,29 +458,42 @@ MuseScore {
     }
 
     function getPossibleChordsList() {
-
-        var T = new HarmonicFunction.HarmonicFunction("T", 1, undefined, "1", [], [],
-                                                      [], false, undefined)
-        var S = new HarmonicFunction.HarmonicFunction("S", 4, undefined, "1", [], [],
-                                                      [], false, undefined)
-        var D = new HarmonicFunction.HarmonicFunction("D", 5, undefined, "1", [], [],
-                                                      [], false, undefined)
-
-        var D7 = new HarmonicFunction.HarmonicFunction("D", 5, undefined, "1", [],
-                                                       ["7"], [], false)
-        var D9 = new HarmonicFunction.HarmonicFunction("D", 5, undefined, "1", [],
-                                                       ["7", "9"], [], false)
-        //todo jak to uzupelnic?
-        var T6 = new HarmonicFunction.HarmonicFunction("T", 6, undefined, "1", [], [],
-                                                       [], false)
-        var S2 = new HarmonicFunction.HarmonicFunction("S", 2, undefined, "1", [], [],
-                                                       [], false)
-        var chopin = new HarmonicFunction.HarmonicFunction("D", 1, undefined, "1", [],
-                                                           [], [], false)
-        var neapol = new HarmonicFunction.HarmonicFunction("S", 1, undefined, "1", [],
-                                                           [], [], false)
-        var S6 = new HarmonicFunction.HarmonicFunction("S", 6, undefined, "1", [], [],
-                                                       [], false)
+        var mode = tab3.item.getSelectedMode()
+        var x = undefined    
+        var T = new HarmonicFunction.HarmonicFunction("T",x,x,x,x,x,x,x,x,mode)
```

Comment:
Nie jestem pewny czy "x" w tych miejscach to dobry pomysł. Sprawdzałeś co z tym zrobi kontruktor hf?

Reply:
Tzn. x to po prostu undefined. Ustawiłem tak, żeby definicje mieściły się w jednej linii.